app/models/language.rb

```diff
@@ -9,6 +9,7 @@ def set_rouge_name
     case downcased_name
     when 'csv' then self.rouge_name = 'csvs'
     when 'html+erb' then self.rouge_name = 'erb'
+    when 'jupyter notebook' then self.rouge_name = 'markdown'
```

Comment:
Don't think there is a formatter for jupyter notebook so using markdown for the moment to prevent crashes